packages/api/src/rx/index.ts

```diff
@@ -77,12 +84,15 @@ const l = logger('api-rx');
  *       ])
  *   )
  *   .subscribe(([blockPeriod, timestamp]) => {
+ *     // Convert the timestamps from type `Moment` into seconds
+ *     const blockPeriodSeconds = moment(blockPeriod).unix();
+ *     const timestampSeconds = moment(timestamp).unix();
  *     const elapsed = last
- *       ? `, ${timestamp.toNumber() - last}s since last`
+ *       ? `, ${timestampSeconds - last}s since last`
```

Comment:
+1 Ahhh, yes, this was pre-change to extending date. (Maybe use `.getTime()` above, it is more along the line what 90% out there use.